homeassistant/components/pyload/sensor.py

```diff
@@ -58,7 +58,7 @@ class PyLoadSensorEntity(StrEnum):
 SENSOR_DESCRIPTIONS: tuple[SensorEntityDescription, ...] = (
     SensorEntityDescription(
         key=PyLoadSensorEntity.SPEED,
-        name="Speed",
+        translation_key=PyLoadSensorEntity.SPEED,
```

Comment:
```suggestion
        translation_key=PyLoadSensorEntity.SPEED.value,
```

Reply:
Oh,well, i already did it like that in several integrations. It is an StrEnum. 🤔

Reply:
It could work but the snapshots look Goofy ah

Reply:
Never has problems with it. In the state machine it stores the object representation,  but it translates to its string value just fine.

Reply:
unit_of_measurement is also stored like this <UnitOfDataRate.MEGABYTES_PER_SECOND: 'MB/s'>